ripe/atlas/cousteau/request.py

```diff
@@ -24,17 +24,14 @@ class AtlasRequest(object):
     }
 
     def __init__(self, **kwargs):
+
         self.url = ""
-        self.url_path = ""
-        self.key = kwargs.get("key", None)
-        if "url_path" in kwargs:
-            self.url_path = kwargs["url_path"]
-        if "server" in kwargs:
-            self.server = kwargs["server"]
-        else:
-            self.server = "atlas.ripe.net"
+        self.key = kwargs.get("key")
+        self.url_path = kwargs.get("url_path", "")
+        self.server = kwargs.get("server", "atlas.ripe.net")
```

Comment:
although it's unlikely to change, I don't like to hard code this kind of infomation.